website/docs/components/code-editor/partials/code/how-to-use.md

```diff
@@ -0,0 +1,88 @@
+## How to use this component
+
+The HDS code editor component is provided as both a `CodeEditor` component and as an `hds-code-editor` Ember [modifier](/components/code-editor?tab=code#ember-modifier). To use this component, you must either include the yielded `Title` component, provide an `@ariaLabel`, or specify an `@ariaLabelledBy`.
+
+The code editor uses [CodeMirror 6](https://codemirror.net/) under the hood.
```

Comment:
```suggestion
```

Reply:
I think we can delete this line here since as @jorytindall commented it's included in the API section.